From 8002a6125bf3f36144a92643ea02ad3abfa5d6d8 Mon Sep 17 00:00:00 2001
From: Jo Zzsi <jozzsicsataban@gmail.com>
Date: Sun, 12 Jan 2025 20:01:09 -0500
Subject: [PATCH 09/13] fix(systemd-ask-password): do not half-install
 systemd-ask-password-wall

Do not install the path unit when the service unit is not installed
for systemd-ask-password-wall.

Fixes the following warning on the CI:

[FAILED] Failed to start Forward Password Requests to Wall Directory Watch.
See 'systemctl status systemd-ask-password-wall.path' for details.
...
systemd[1]: systemd-ask-password-wall.path: Refusing to start, unit systemd-ask-password-wall.service to trigger not loaded.
systemd[1]: Failed to start Forward Password Requests to Wall Directory Watch.

(cherry picked from commit 4ddc0053e869eb37b7c3d4e08876a687e1a995ae)

Related: RHEL-65204
---
 modules.d/01systemd-ask-password/module-setup.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/modules.d/01systemd-ask-password/module-setup.sh b/modules.d/01systemd-ask-password/module-setup.sh
index 40774ab9..8b09b69f 100755
--- a/modules.d/01systemd-ask-password/module-setup.sh
+++ b/modules.d/01systemd-ask-password/module-setup.sh
@@ -43,7 +43,6 @@ install() {
     inst_multiple -o \
         "$systemdsystemunitdir"/systemd-ask-password-console.path \
         "$systemdsystemunitdir"/systemd-ask-password-console.service \
-        "$systemdsystemunitdir"/multi-user.target.wants/systemd-ask-password-wall.path \
         "$systemdsystemunitdir"/sysinit.target.wants/systemd-ask-password-console.path \
         systemd-ask-password \
         systemd-tty-ask-password-agent
-- 
2.47.1

